modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/detailedcircuit/v0.0.1.json

```diff
@@ -0,0 +1,56 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "this": "{{base}}/schemas/bbp/simulation/detailedcircuit/v0.0.1/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/hbphackaton/simulation/modelinstance/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:DetailedCircuitShape",
+      "@type": "sh:NodeShape",
+      "targetClass": "nsg:DetailedCircuit",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/hbphackaton/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:nodeCollection",
+              "name": "Node collection",
+              "description": "Node collection entity.",
+              "seeAlso": "{{base}}/schemas/bbp/simulation/nodecollection/v0.1.0/shapes/NodeCollectionShape",
+              "class": "nsg:NodeCollection",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:edgeCollection",
+              "name": "nrnPath",
```

Comment:
Nrn path

Reply:
I don't like nrn path. This is implementation detail and subject to change.
"edge" only ?